api/framer-motion.api.md

```diff
@@ -445,7 +445,7 @@ export class MotionValue<V = any> {
     isAnimating(): boolean;
     onChange(subscription: Subscriber<V>): () => void;
     // @internal
-    onRenderRequest(subscription: Subscriber<V>): () => boolean;
+    onRenderRequest(subscription: Subscriber<V>): () => undefined;

```

Comment:
It is returning a function

```
: () => undefined;
```

The change from boolean to undefined is because we're explicitly voiding what was before a boolean (Set.delete)

Reply:
Ah, I misread this type 🤦